geozero/src/wkt/wkt_reader.rs

```diff
@@ -107,15 +107,14 @@ fn process_point<P: GeomProcessor>(
     idx: usize,
     processor: &mut P,
 ) -> Result<()> {
-    processor.point_begin(idx)?;
     if let Some(ref coord) = point.0 {
+        processor.point_begin(idx)?;
         process_coord(coord, multi_dim, 0, processor)?;
+        processor.point_end(idx)
     } else {
-        return Err(GeozeroError::Geometry(
-            "The WKT Point was empty, but Points cannot be empty in other formats".to_string(),
-        ));
+        // skip processing of POINT EMPTY for now since no other formats support it
```

Comment:
Is
```rust
    processor.point_begin(idx)?;
    if let Some(ref coord) = point.0 {
        process_coord(coord, multi_dim, 0, processor)?;
    } else {
        // POINT EMPTY
    }
    processor.point_end(idx)
```
giving problems? It looks like the correct representation of an empty geometry to me.

Reply:
I think the spelling is `POINT EMPTY` not `POINT ()`